components/selectDevice/Device.tsx

```diff
@@ -0,0 +1,53 @@
+"use client";
+
+import { useRouter } from "next/navigation";
+import { useState, useTransition } from "react";
+
+import transferPlayback from "@/lib/device/transferPlayback";
+
+const Device = ({
+  device,
+  currentDeviceId,
+  isPlaying,
+}: {
+  device: SpotifyApi.UserDevice;
+  currentDeviceId: string;
+  isPlaying: boolean;
+}) => {
+  const router = useRouter();
+  const [isPending, startTransition] = useTransition();
+  const [isFetching, setIsFetching] = useState(false);
+
+  // Create inline loading UI
+  const isMutating = isFetching || isPending;
+
+  const handleClick = async (e: React.MouseEvent) => {
```

Comment:
Can you wrap this whole thing in a toast promise? Go here:
https://react-hot-toast.com/docs/toast 

and it's under Promise. Something like `success: Transferred to ${device.name}`?

Reply:
actually nvm we don't need the promise wrapper (especially if you just use the isFetching call), just do a toast on success